src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -5305,6 +5501,8 @@ subroutine atm_compute_dyn_tend_work(nCells, nEdges, nVertices, nVertLevels_dumm
                   
                end do
             end do
+
```

Comment:
Seems like an extra newline

Reply:
Generally, I was trying to make the `parallel` regions stand out, and so it was a deliberate choice to leave a blank line between the `end do` and the `!$acc end parallel` directive.

Reply:
For consistency, though, I'll remove this blank line.